content/developer/docker.md

```diff
@@ -28,7 +28,7 @@ Switch DBMS
 By default, wallabag starts with a SQLite database. However, wallabag
 also supports PostgreSQL and MySQL. Docker containers are available for both.
 
-In `docker-compose.yml`, for the chosen DBMS uncomment:
+In `compose.yaml`, for the chosen DBMS uncomment:
```

Comment:
Only available from 2.6.11
https://github.com/wallabag/wallabag/blob/2.6.10/docker-compose.yml

Reply:
Shouldn't we mention it?